bazel/README.md

```diff
@@ -55,6 +55,16 @@ _note_: `coreutils` is used for realpath
 Envoy compiles and passes tests with the version of clang installed by XCode 9.3.0:
 Apple LLVM version 9.1.0 (clang-902.0.30).
 
+In order for bazel to be aware of the tools installed by brew, the PATH
+variable must be set for bazel builds. This can be accomplished setting
+
+```
+--action_env=PATH=/bin:/opt/local/bin:/usr/bin:/usr/local/bin"
+```
+
+either on the command line when running `bazel build`/`bazel test` or
+in your `bazelrc` file.
```

Comment:
Nit: It's `.bazelrc`.

We could probably be more specific and say `$HOME/.bazelrc`. But it's fine if you want to omit `$HOME` part.

Reply:
Being explicit makes sense to me here, so I'll do the full thing